index.html

```diff
@@ -97,53 +102,24 @@ <h2>Multi-Post Stories</h2>
               </ul>

```

Comment:
### Your HomePage
![homepage (copy)](https://user-images.githubusercontent.com/93039359/179304271-56c3d4b1-2da0-49b5-b1c3-8f7c1b6849f3.png)

### Figma HomePage
![figma-homepage (copy)](https://user-images.githubusercontent.com/93039359/179304395-016468fa-750f-4446-a23c-c45f7f7ba9e9.png)

- [ ] Your title is perfectly aligned but your paragraph is not perfectly aligned. Please align it according to the Figma template. :+1: